sdk/spring/tests.yml

```diff
@@ -96,76 +92,39 @@ stages:
-              AAD_B2C_USER_PASSWORD: $(java-spring-aad-b2c-user-password)
-              AAD_B2C_CLIENT_ID: $(java-spring-aad-b2c-client-id)
-              AAD_B2C_CLIENT_SECRET: $(java-spring-aad-b2c-client-secret)
-              AAD_USER_NAME_ON_DEMAND: ${{ parameters.AadOnDemandUserName }}
-              AAD_USER_PASSWORD_ON_DEMAND: $(java-spring-aad-user-password-1)
-              AAD_SERVICE_PRICIPAL_CLIENT_ID: $(java-spring-aad-service-principal-client-id)
-              AAD_SERVICE_PRICIPAL_CLIENT_SECRET: $(java-spring-aad-service-principal-client-secret)
+    - template: spring-test-template.yml
+      parameters:
+        AadOnDemandUserDisplayName: ${{ parameters.AadOnDemandUserDisplayName }}
+        TestName: LiveTestInAzureChina
+        Clouds: China
+        TestResourceDirectories:
+          - spring/azure-spring-boot-test-aad
+        Artifacts:
+          - name: azure-spring-boot-test-core
+            groupId: com.azure.spring
+            safeName: azurespringboottestcore
+          - name: azure-spring-boot-test-aad
+            groupId: com.azure.spring
+            safeName: azurespringboottestaad
 
-          - powershell: |
-              az login --allow-no-subscriptions --tenant $(java-spring-aad-tenant-id-1) --service-principal -u $(java-spring-aad-service-principal-client-id) -p $(java-spring-aad-service-principal-client-secret)
-              az ad user delete --id "${{ parameters.AadOnDemandUserName }}"
-              az logout
-            condition: always()
-            displayName: 'Delete On-demand test user'
+        EnvVars:
+          AZURE_TEST_MODE: LIVE
+          AAD_TENANT_ID_1: $(java-spring-aad-tenant-id-1-cn)
+          AAD_USER_NAME_1: $(java-spring-aad-user-name-1-cn)
+          AAD_USER_PASSWORD_1: $(java-spring-aad-user-password-1-cn)
+          AAD_TENANT_ID_2: $(java-spring-aad-tenant-id-2-cn)
+          AAD_USER_NAME_2: $(java-spring-aad-user-name-2-cn)
+          AAD_USER_PASSWORD_2: $(java-spring-aad-user-password-2-cn)
+          AAD_MULTI_TENANT_CLIENT_ID: $(java-spring-aad-multi-tenant-client-id-cn)
+          AAD_MULTI_TENANT_CLIENT_SECRET: $(java-spring-aad-multi-tenant-client-secret-cn)
+          AAD_SINGLE_TENANT_CLIENT_ID: $(java-spring-aad-single-tenant-id-cn)
+          AAD_SINGLE_TENANT_CLIENT_SECRET: $(java-spring-aad-single-tenant-secret-cn)
+          AAD_SINGLE_TENANT_CLIENT_ID_WITH_ROLE: $(java-spring-aad-single-tenant-id-with-role-cn)
+          AAD_SINGLE_TENANT_CLIENT_SECRET_WITH_ROLE: $(java-spring-aad-single-tenant-secret-with-role-cn)
+          AAD_USER_NAME_ON_DEMAND: ${{ parameters.AadOnDemandUserNameChina }}
+          AAD_USER_PASSWORD_ON_DEMAND: $(java-spring-aad-user-password-1-cn)
+          AAD_SERVICE_PRICIPAL_CLIENT_ID: $(java-spring-aad-service-principal-client-id-cn)
+          AAD_SERVICE_PRICIPAL_CLIENT_SECRET: $(java-spring-aad-service-principal-client-secret-cn)
+          AZURE_CLOUD_TYPE: "China"
```

Comment:
Can we just keep `TestName, Clouds, TestResourceDirectories, Artifacts` in `tests.yml`, and move all other contents into `spring-test-template.yml`?